base/sysimg.jl

```diff
@@ -473,6 +469,7 @@ unshift!(Base._included_files, (@__MODULE__, joinpath(@__DIR__, "sysimg.jl")))
 # load some stdlib packages but don't put their names in Main
 Base.require(:DelimitedFiles)
 Base.require(:Test)
+Base.require(:Dates)
```

Comment:
What decides if  an stdlib package should be required here or not?

Reply:
Anyone know the answer here? @JeffBezanson?

Reply:
These are to reduce startup/load time.